Description: Use sbin for ntploggps and ntplogtemp
 We install ntploggps and ntplogtemp to /usr/sbin, so the man pages should be
 in section 8, not section 1.
Forwarded: not-needed
Origin: vendor
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2018-08-31
--- a/ntpclients/ntploggps-man.adoc
+++ b/ntpclients/ntploggps-man.adoc
@@ -1,4 +1,4 @@
-= ntploggps(1) =
+= ntploggps(8) =
 :man version: @NTPSEC_VERSION@
 include::../docs/manual.include[]
 
--- a/ntpclients/ntplogtemp-man.adoc
+++ b/ntpclients/ntplogtemp-man.adoc
@@ -1,4 +1,4 @@
-= ntplogtemp(1) =
+= ntplogtemp(8) =
 :man version: @NTPSEC_VERSION@
 include::../docs/manual.include[]
 
--- a/wscript
+++ b/wscript
@@ -1063,7 +1063,7 @@
         afterparty(ctx)
 
     if ctx.env['PYTHON_GPS']:
-        ctx.manpage(1, "ntpclients/ntploggps-man.adoc")
+        ctx.manpage(8, "ntpclients/ntploggps-man.adoc")
     ctx.manpage(1, "ntpclients/ntpdig-man.adoc")
     if ctx.env['PYTHON_CURSES']:
         ctx.manpage(1, "ntpclients/ntpmon-man.adoc")
@@ -1071,7 +1071,7 @@
     ctx.manpage(1, "ntpclients/ntpsweep-man.adoc")
     ctx.manpage(1, "ntpclients/ntptrace-man.adoc")
     ctx.manpage(1, "ntpclients/ntpviz-man.adoc")
-    ctx.manpage(1, "ntpclients/ntplogtemp-man.adoc")
+    ctx.manpage(8, "ntpclients/ntplogtemp-man.adoc")
     ctx.manpage(8, "ntpclients/ntpkeygen-man.adoc")
     ctx.manpage(8, "ntpclients/ntpleapfetch-man.adoc")
     ctx.manpage(8, "ntpclients/ntpwait-man.adoc")
--- a/etc/ntploggps.service
+++ b/etc/ntploggps.service
@@ -1,9 +1,9 @@
 [Unit]
-Documentation=man:ntploggps(1)
+Documentation=man:ntploggps(8)
 Description=GPS information logger for ntpviz
 Requisite=gpsd.service
 After=gpsd.service
 
 [Service]
 Type=simple
-ExecStart=/usr/bin/ntploggps -o -l /var/log/ntpsec/gpsd
+ExecStart=/usr/sbin/ntploggps -o -l /var/log/ntpsec/gpsd
--- a/etc/ntplogtemp.service
+++ b/etc/ntplogtemp.service
@@ -1,7 +1,7 @@
 [Unit]
-Documentation=man:ntplogtemp(1)
+Documentation=man:ntplogtemp(8)
 Description=Temperature information logger for ntpviz
 
 [Service]
 Type=simple
-ExecStart=/usr/bin/ntplogtemp -o -l /var/log/ntpsec/temps
+ExecStart=/usr/sbin/ntplogtemp -o -l /var/log/ntpsec/temps
